src/server/locale/US/PAY_LATER_LONG_TERM/mutations/index.js

```diff
@@ -7,12 +7,14 @@ import lt_mq from './lt_mq';
 import lt_mqez from './lt_mqez';
 import lt_sq from './lt_sq';
 import lt_sqez from './lt_sqez';
+import lt_no_amnt from './lt_no_amnt';
 
 export default function getMutations(id, type) {
     switch (id) {
-        case 'LT_NQ_MIN':
-        case 'LT_NQ_MAX':
+        case 'LT_NQ':
             return lt_nq[type];
+        case 'LT_NO_AMNT':
+            return lt_no_amnt[type];
         case 'LT_NQEZ_MIN':
         case 'LT_NQEZ_MAX':
```

Comment:
```suggestion
        case 'LT_NQEZ':
```
Let's replace the two LT_NQEZ_MIN (and MAX) with just `LT_NQEZ` since there's just one now.